### PR TITLE
KN-35784 Add GetAllUnits method to Knoema.Client

### PR DESCRIPTION
**Pull request by ketiketi2507**

### CODE REVIEW

File: Knoem.Client.UnitTests/KnoemaClientTests.cs [hunk 1]
```diff
@@ -21,15 +21,32 @@ public void GetTimeSeriesList()
 				DimensionId = "country",
 				Members = { 1000000, 1000100 }
 			});
-			request.DimensionRequest.Add( new DimensionRequestItem()
+			request.DimensionRequest.Add(new DimensionRequestItem()
```
Comment (unicrus) [inline Knoem.Client.UnitTests/KnoemaClientTests.cs]: No need in brackets in this case.

File: Knoema.Client/Client.cs [hunk 2]
```diff
@@ -175,12 +175,17 @@ public Task<FlatTimeSeriesRawDataResponse> GetFlatDataStreaming(string token)
             return ApiGet<FlatTimeSeriesRawDataResponse>("/api/1.0/data/raw/", string.Format("continuationToken={0}", token));
         }
 
+		public Task<IEnumerable<UnitMember>> GetAllUnits()
```
Comment (PavelPimenov) [inline Knoema.Client/Client.cs]: Please rename method to GetUnits

File: Knoema.Client/Client.cs [hunk 3]
```diff
@@ -175,12 +175,17 @@ public Task<FlatTimeSeriesRawDataResponse> GetFlatDataStreaming(string token)
...
+			return ApiGet<IEnumerable<UnitMember>>("/api/1.0/meta/units");
+		}
+
 		public Task<IEnumerable<TimeSeriesItem>> GetTimeSeriesList(string datasetId, FullDimensionRequest request)
 		{
 			return ApiPost<IEnumerable<TimeSeriesItem>>(string.Format("/api/1.0/data/dataset/{0}", datasetId), request);
 		}
 
-        public async Task<PostResult> UploadPost(string fileName)
+		public async Task<PostResult> UploadPost(string fileName)
```
Comment (AlexPhilippov) [inline Knoema.Client/Client.cs]: Format

File: Knoem.Client.UnitTests/KnoemaClientTests.cs (hunk 1 above)
Comment (ketiketi2507) [reply Knoem.Client.UnitTests/KnoemaClientTests.cs]: fixed

File: Knoema.Client/Client.cs (hunk 2 above)
Comment (ketiketi2507) [reply Knoema.Client/Client.cs]: fixed

File: Knoema.Client/Client.cs (hunk 3 above)
Comment (ketiketi2507) [reply Knoema.Client/Client.cs]: fixed

File: Knoema.Client/Client.cs [hunk 4]
```diff
@@ -175,12 +175,17 @@ public Task<FlatTimeSeriesRawDataResponse> GetFlatDataStreaming(string token)
             return ApiGet<FlatTimeSeriesRawDataResponse>("/api/1.0/data/raw/", string.Format("continuationToken={0}", token));
         }
 
+		public Task<IEnumerable<UnitMember>> GetUnits()
```
[the remark this answers is not in the document]
Comment (ketiketi2507) [reply Knoema.Client/Client.cs]: So it was in the specification

**Pull request merged**